tests/e2e/files/devfiles/plugins/VscodeValePlugin.yaml

```diff
@@ -6,7 +6,7 @@ projects:
     source:
       location: 'https://github.com/eclipse/che-docs.git'
       type: git
-      branch: master
+      branch: che-qe
```

Comment:
What is the reason for changing branch here? Will it require some manual updates for the branch to keep it updated?

Reply:
First of all, this is just a sample and it doesn't relate directly to the plugin functionality. I want to have a persistent sample version without the risk that something will be changed in the documentation and the test will fail. Besides in this branch, I deleted all "README" files, which significantly simplify testing and avoid unexpected openings not expected files.

And no, it doesn't require manual updates. I plan to have one sample version.